app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java

```diff
@@ -18,11 +31,120 @@
 
     public static final String TAG = HomeFragment.class.getSimpleName();
 
+    //retrieved from the backend, more specifically from the Person's list of Lao's
+    private ArrayList<Lao> laos;
+    //Now: for testing, later: retrieve from Person
```

Comment:
Maybe just add a TODO tag so we remember to change that

Reply:
done